data_prep/introspector.py

```diff
@@ -42,7 +42,11 @@
 def _query_introspector(api: str, params: dict) -> dict:
   """Queries FuzzIntrospector API and return data specified by |key|,
   returns None if unable to get the value."""
-  resp = requests.get(api, params, timeout=TIMEOUT)
+  try:
```

Comment:
Thanks for addressing this so quickly, @cjx10!

Just a minor tip: It would be nicer to have this in a different PR, though : ) This is different from the `RESOURCE_EXHAUSTED` error in the title.
But no need to change it this time.